[export] Move swap to a different file

angelayi opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #137134
* #137102
* #136191

Refactor so that unflattener doesn't become too messy

Differential Revision: [D63719648](https://our.internmc.facebook.com/intern/diff/D63719648/)

angelayi committed:
[export] Move swap to a different file

Refactor so that unflattener doesn't become too messy

Differential Revision: [D63719648](https://our.internmc.facebook.com/intern/diff/D63719648/)

[ghstack-poisoned]

avikchaudhuri reviewed (approved):
What's the status of being able to import things from `torch.export.unflatten`? Since it is both a module as well as a function...

angelayi commented:
> What's the status of being able to import things from torch.export.unflatten? Since it is both a module as well as a function...

I have no clue.. I don't really know how that works..

angelayi committed:
Update on "[export] Move swap to a different file"


Refactor so that unflattener doesn't become too messy

Differential Revision: [D63719648](https://our.internmc.facebook.com/intern/diff/D63719648/)

[ghstack-poisoned]

angelayi committed:
Update on "[export] Move swap to a different file"


Refactor so that unflattener doesn't become too messy

Differential Revision: [D63719648](https://our.internmc.facebook.com/intern/diff/D63719648/)

[ghstack-poisoned]
